docs/quick-start/index.mdx

```diff
@@ -16,9 +16,22 @@ import PuppeteerSource from '!!raw-loader!./quick_start_puppeteer.ts';
 
 import CheerioLog from '!!raw-loader!./quick_start_cheerio.txt';
 
-With this short tutorial you can start scraping with Crawlee in a minute or two. To learn how Crawlee works, read the [Introduction](./introduction), which is a comprehensive step-by-step guide for creating your first scraper.
+With this short tutorial you can start scraping with Crawlee in a minute or two. To learn in-depth how Crawlee works, read the [Introduction](./introduction), which is a comprehensive step-by-step guide for creating your first scraper.
 
-## Local stand-alone usage
+## Choose your crawler
+
+Crawlee comes with three different crawler classes: <ApiLink to="cheerio-crawler/class/CheerioCrawler">`CheerioCrawler`</ApiLink>, <ApiLink to="puppeteer-crawler/class/PuppeteerCrawler">`PuppeteerCrawler`</ApiLink> and <ApiLink to="playwright-crawler/class/PlaywrightCrawler">`PlaywrightCrawler`</ApiLink>. All classes share the same interface for maximum flexibility when switching between them.
```

Comment:
I wanted to skip it, because it's really not something that you should use in a quick-start. Plus we don't have examples for it. Every time someone uses basic crawler, it adds complexity to the project 😄